Sources/Kevin/InAppPayments/PaymentConfirmation/KevinPaymentConfirmationViewModel.swift

```diff
@@ -25,13 +25,7 @@ internal class KevinPaymentConfirmationViewModel : KevinViewModel<KevinPaymentCo
     
     private func initialize(_ configuration: KevinPaymentConfirmationConfiguration) {
         var confirmationUrlString: String!
-        if configuration.paymentType == .card {
-            confirmationUrlString =  String(
-                format: KevinApiPaths.cardPaymentUrl,
-                configuration.paymentId,
-                Kevin.shared.locale.identifier.lowercased()
-            )
-        } else if configuration.paymentType == .bank {
+        if configuration.paymentType == .bank {
```

Comment:
Do we even need this check? Payment type has only one option now so there only one possible flow/scenario

Reply:
Yeah, probably also makes sense to remove it 👍